src/app/components/shared/transforming-radio-group-collection/TransformingRadioGroupCollection.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import TransformingRadioGroup from './../transforming-radio-group/TransformingRadioGroup';
+
+export default class TransformingRadioGroupCollection extends Component {
+	componentWillMount() {
```

Comment:
Skal komponenten være dynamisk slik at det å sette opp lokal state også må håndteres når props endres? Klarer vi å løse denne komponenten uten intern state?

Reply:
Litt usikker på om jeg skjønner spørsmålet. Forstår jeg det riktig som at du lurer på om man kan gå vekk fra å ha callback-funksjoner (handleOnClickCollapsed/Expanded) med state-håndtering lokalt i komponentene som bruker `TransformingRadioGroupCollection`, til fordel for å bare sende inn et state-objekt fra komponentene som bruker den direkte og gjøre state-håndteringen i `TransformingRadioGroupCollection`?